tests/functional/test_e2e.py

```diff
@@ -129,19 +147,73 @@ def test_e2e_release_files(page, live_server, dev_users, release_files_stubber):
     expect(filegroup_link).to_contain_text(re.compile("my-new-group", flags=re.I))
 
     # In the initial request view, the tree is collapsed
-    file_link = page.get_by_role("link").locator(".file:scope")
+    file_link = page.get_by_role("link").locator(".output.file:scope")
     assert file_link.all() == []
 
     # Click to open the filegroup tree
     filegroup_link.click()
 
-    # Click on the directory to ensure that renders correctly.
+    # Click on the output directory to ensure that renders correctly.
+    subdir_link = page.get_by_role("link").locator(".directory:scope")
+    find_and_click(subdir_link)
+    expect(page.locator("#selected-contents")).to_contain_text("file.txt")
+
+    # Tree opens fully expanded, so now the file (in its subdir) is visible
+    find_and_click(file_link)
+    expect(page.locator("body")).to_contain_text("I am the file content")
+    assert "output" in file_link.get_attribute("class")
+
+    # Go back to the Workspace view so we can add a supporting file
```

Comment:
It's there (specifically the returning to add a supporting file) because of the previous way I implemented this (with separate trees for output and supporting files) - so it was checking there were no supporting files shown first). However, we do still want to ensure that we can go back to the workspace view from the request view at least once, to ensure the workspace button is working. Going back to the workspace from the request was already part of the test, I just changed it to do the supporting file there.

Reply:
Ah, ok, yeah, I hadn't though of testing the workspace home link, fair enough